hack/e2e.sh

```diff
@@ -112,19 +112,37 @@ done
 # Wait for cincinnati metrics to be recorded
 # Find out the token Prometheus uses from its serviceaccount secrets
 # and use it to query for GB build info
-export PROM_ENDPOINT=$(oc -n openshift-monitoring get route thanos-querier -o jsonpath="{.spec.host}")
+PROM_ROUTE=$(oc -n openshift-monitoring get route thanos-querier -o jsonpath="{.spec.host}")
+export PROM_ENDPOINT="https://${PROM_ROUTE}"
+echo "Using Prometheus endpoint ${PROM_ENDPOINT}"
+
 export PROM_TOKEN=$(oc -n openshift-monitoring get secret \
   $(oc -n openshift-monitoring get serviceaccount prometheus-k8s \
     -o jsonpath='{range .secrets[*]}{.name}{"\n"}{end}' | grep prometheus-k8s-token) \
   -o go-template='{{.data.token | base64decode}}')
 
 DELAY=30
 for i in $(seq 1 10); do
-  PROM_OUTPUT=$(curl -kLs -H "Authorization: Bearer ${PROM_TOKEN}" "https://${PROM_ENDPOINT}/api/v1/query?query=cincinnati_gb_build_info" || continue)
+  PROM_OUTPUT=$(curl -kLs -H "Authorization: Bearer ${PROM_TOKEN}" "${PROM_ENDPOINT}/api/v1/query?query=cincinnati_gb_build_info" || continue)
   grep "metric" <<< "${PROM_OUTPUT}" && break
 
   sleep ${DELAY}
 done
 
-# Run e2e tests in sequential mode to have SLO tests in the end
-env RUST_TEST_TASKS=1 /usr/bin/cincinnati-e2e-test
+# Show test failure details
+export RUST_BACKTRACE="1"
+
+# Run e2e tests
+/usr/bin/cincinnati-e2e-test
+
+# Ensure prometheus_query tests are executed
+/usr/bin/cincinnati-prometheus_query-test
+
+# Run load-testing script
+bash /usr/local/bin/load-testing.sh
```

Comment:
* Why the extra `bash` here?
* Can the load testing fail somehow?

Reply:
* Removed extra bash as `load-testing.sh` is not executable
* It may fail only if `vegeta` doesn't get installed. In other cases `vegeta report` would return 0, even if there is no data set